diff --git a/node_modules/@mapbox/geojson-coords/index.js b/node_modules/@mapbox/geojson-coords/index.js
index b6bd632..561f68d 100644
--- a/node_modules/@mapbox/geojson-coords/index.js
+++ b/node_modules/@mapbox/geojson-coords/index.js
@@ -4,7 +4,7 @@ var geojsonNormalize = require('@mapbox/geojson-normalize'),
 
 module.exports = function(_) {
     if (!_) return [];
-    var normalized = geojsonFlatten(geojsonNormalize(_)),
+    var normalized = geojsonFlatten.default(geojsonNormalize(_)),
         coordinates = [];
     normalized.features.forEach(function(feature) {
         if (!feature.geometry) return;
